packages/core-snapshots-cli/lib/db/queries/blocks/height-range.sql

```diff
@@ -0,0 +1,20 @@
+SELECT
+  ID,
```

Comment:
Properties in queries should be all lower-case, only SQL keywords should be uppercase.